x/emissions/proto/emissions/v1/types.proto

```diff
@@ -98,18 +113,22 @@ message ValueBundle {
 
   uint64 topic_id = 1;
   bytes extra_data = 2;
-  // L_im or I_i
+  // R_im || log10 L_im || I_i
   double combined_value = 3;
-  // L_ijm or I_ij
+  // R_ijm || log10 L_ijm || I_ij
   repeated WorkerAttributedValue inferer_values = 4;
-  // L_ikm or I_ik
+  // R_ikm || log10 L_ikm || I_ik
   repeated WorkerAttributedValue forecaster_values = 5;
-  // L^-_im or I^-_i
+  // R^-_im  || log10 L^-_im || I^-_i
   double naive_value = 6;
-  // L^-_lim or I^-_li
-  repeated WorkerAttributedValue one_out_values = 7;
-  // L^+_kim or I^+_ki
-  repeated WorkerAttributedValue one_in_naive_values = 8;
+  // R^-_ilkm || log10 L^-_ilkm || I^-_ilk where l = any j
+  // An extra index is necessary because forecast-implied inferences are recomputed for each left-out inference
+  repeated WorkerAttributedValuesOfWithheldWorker one_out_inferer_values = 7;
+  // R^-_ilm || log10 L^-_ilm || I^-_il where l = any k
+  repeated WithheldWorkerAttributedValue one_out_forecaster_values = 8;
```

Comment:
What `L^-_ilkm` mean? 

and I guess it's the `one_out_forecaster_values` that needs to have `WorkerAttributedValuesOfWithheldWorker` type.

Reply:
Also, considering diederik comment in the gc: 
```
reputer combinator loss oneout
L^-_lim where l = some j or k (l goes from 1 to n_inferences + n_forecasts,  j goes from 1 to n_inferences , and k from 1 to n_forecasts)
````

Prob we should use a new type for `one_out_inferer_values` since it will be an array that includes both withheld workers inference losses and forecast losses. Perhaps it should be named `one_out_values`.

Reply:
talked ab this on call. remedied in later commit.